packages/datatrak-web/src/features/Questions/EntityQuestion.tsx

```diff
@@ -7,19 +7,30 @@ import { EntitySelector } from '../EntitySelector';
 export const EntityQuestion = ({
   id,
   label,
+  detailLabel,
   name,
   required,
   controllerProps: { onChange, value, ref, invalid },
   config,
 }: SurveyQuestionInputProps) => {
-  const { isReviewScreen, isResponseScreen, formData, countryCode } = useSurveyForm();
-  const { surveyProjectCode } = useSurveyForm();
+  const {
+    isReviewScreen,
+    isResponseScreen,
+    formData,
+    countryCode,
+    displayQuestions,
+    screenHeader,
+    surveyProjectCode,
+  } = useSurveyForm();
+
+  // Hide the question label if there is only one question, and it is the same as the screen header
+  const hideQuestionLabel = displayQuestions.length === 1 && screenHeader == label;
 
   return (
     <EntitySelector
       id={id}
-      label={label}
-      detailLabel={`Select an entity from the list below ${required ? '*' : ''}`}
+      label={hideQuestionLabel ? '' : label}
```

Comment:
Assuming it’s nullable, this feels a touch more semantically appropriate?

```suggestion
      label={hideQuestionLabel ? null : label}
```